video_xblock/backends/base.py

```diff
@@ -106,6 +106,15 @@ def metadata_fields(self):
         return []
 
     @property
+    def editable_fields(self):
+        """
+        Tuple of all editable VideoXBlock fields to be validated.
+
+        Defaults to concatenation of `basic_fields` and `advanced_fields`.
+        """
+        return tuple(itertools.chain(self.basic_fields, self.advanced_fields))
```

Comment:
Well, this solution will require converting two tuples into sets. Althought, my bet is we're not going to notice a performance difference with the data we use.

As for readability, both options look similar to me.

Also, itertools is part of Python's stdlib.